scripts/setup-host-macos.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+echo "Nothing to install, OSVM should contain SpellingPlugin"
+
+echo "Let's install ispell anyway so we can test mac OS callouts from IspellSpellingInterface, just in case the SpellingPlugin is not available ..."
```

Comment:
```suggestion
echo "But let's install ispell anyway so we can test mac OS callouts from IspellSpellingInterface, just in case the SpellingPlugin is not available ..."
```